src/ucp/core/ucp_context.c

```diff
@@ -227,6 +227,11 @@ static ucs_config_field_t ucp_config_table[] = {
    "to ucp_init()",
    ucs_offsetof(ucp_config_t, ctx.estimated_num_eps), UCS_CONFIG_TYPE_ULUNITS},
 
+  {"NUM_PPN", "auto",
+   "An optimization hint for single node. Does not affect semantics, only transport\n"
```

Comment:
Since it is "help" message it has to be a bit more descriptive - Process Per Node (PPN) hint for optimization shared memory performance. @tonycurtis probably will word this better

Reply:
updated